dev-env-app/server/src/startDevEnv.ts

```diff
@@ -95,9 +95,15 @@ export function startDevEnvPlugin({ setBehodlerDevEnv, setStartDevEnv }): Fastif
           fs.rmSync(DEPLOYMENT_CONFIG.addressesJSONFilePath)
           fastify.log.info('file removed, starting dev env')
         }
-
-        setBehodlerDevEnv(await startHardhatNodeAndDeployBehodlerContracts())
+        const behodlerDevEnv:BehodlerDevEnv = await startHardhatNodeAndDeployBehodlerContracts()
+        setBehodlerDevEnv(behodlerDevEnv)
         fastify.log.info('dev env started')
+        fastify.log.info("listening on port " + HRE_NODE_SETTINGS.port)
+        const { chainId } = await hre.ethers.provider.getNetwork()
+        fastify.log.info('chainId ' + chainId)
+        const [deployer] = await hre.ethers.getSigners();
+        fastify.log.info('Deployer account ' + deployer.address)
+        fastify.log.info(JSON.stringify(behodlerDevEnv.deployedAddresses,null,4))
```

Comment:
I would move this before logging the RPC server port. The map is long and would obfuscate to rest of the logged information.

Reply:
Better yet, remove it entirely, given that we have an endpoint for this. It's very un-Unixy to log something that can be fetched with a GET request.